### DISCUSSION TITLE
Can Fairlearn already deal with regression problems?

### DISCUSSION THREAD

**Discussion by bspanjers** [fairlearn/fairlearn#387] [Q&A] [answered]
Hi,

First of all: thank you very much for making this repository. I am deeply interested in algorithmic fairness and I am currently working on my BSc thesis about fair regression. My question is, since I cannot find a notebook about a regression problem, if the algorithms already work for regression problems. Furthermore, I wonder if I can expect an algorithm for fair regression under statistical parity in the future as well. 

Thanks in advance.

Barend

**Comment by srayagarwal** [comment]
Yes, it can. Try additive counterfactual fairness method

**Comment by riedgar-ms** [comment, marked as answer]
You can run the `GridSearch` algorithm with bounded group loss for regression. There is also work underway to have regression fully supported in `ExponentiatedGradient`.

**Comment by romanlutz** [comment]
Thanks for your question @bspanjers !

+1 on @riedgar-ms 's statement.

We're actively working on adding a notebook/example to illustrate how to use `GridSearch` with `BoundedGroupLoss`, but it should be straightforward and just like the classification examples. Please reach out if you have questions/issues. In general, we're more than happy to answer usage questions in our [Gitter community](https://gitter.im/fairlearn/community). Feel free to join! If there's anything you learn about fair regression during your studies that you'd like to share (e.g. "Fairlearn is missing X!", "I have an implementation of Y that Fairlearn could add", etc.) we're definitely interested.

Additive Counterfactual Fairness (as referenced in the first response) isn't part of Fairlearn at this point.

Closing this issue since it's a usage question.
